mne/io/kit/kit.py

```diff
@@ -90,10 +90,9 @@ def __init__(self, input_fname, mrk=None, elp=None, hsp=None, stim='>',
         # meg channels plus synthetic channel
         info['nchan'] = self._sqd_params['nchan'] + 1
 
-        cals = np.ones(info['nchan'])
         last_samps = [self._sqd_params['nsamples'] - 1]
         super(RawKIT, self).__init__(
-            info, cals=cals, last_samps=last_samps, verbose=verbose)
+            info, cals=None, last_samps=last_samps, verbose=verbose)
```

Comment:
If you are unifying the computation, the `_BaseRaw` init probably doesn't even need a `cals` parameter anymore.